monai/bundle/scripts.py

```diff
@@ -66,28 +66,35 @@
 PPRINT_CONFIG_N = 5
 
 
-def _update_args(args: str | dict | None = None, ignore_none: bool = True, **kwargs: Any) -> dict:
+def update_kwargs(args: str | dict | None = None, ignore_none: bool = True, **kwargs: Any) -> dict:
```

Comment:
Do we need to support sequence str here and use `load_config_files` in L89?

Reply:
good point, that logic in in the config parser https://github.com/Project-MONAI/MONAI/blob/fc1350aae1de702404189f334ac17fd6f51321d0/monai/bundle/config_parser.py#L318-L320

but that reminds me there's a case of comma separated filenames https://github.com/Project-MONAI/research-contributions/blob/b2809b5adfa49b7e9220f1e2d1ff26a16f23dc1b/auto3dseg/algorithm_templates/swinunetr/scripts/train.py#L142-L143 I'll update the load config to fix it

Reply:
had another look and add support for sequence str, which can close https://github.com/Project-MONAI/MONAI/issues/5899 at the CLI level.